db/migrate/20171124131334_create_certificates.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCertificates < ActiveRecord::Migration
+  def change
+    create_table :certificates do |t|
+      t.text :certificate
+      t.date :expiration
```

Comment:
How are we going to extract this information? AFAIK we are not doing that yet.

Reply:
good point, it's not required for now, but it can be set later through the UI i would say. we could also make this mandatory already now

Reply:
I would drop it because we are not using it.

I would envision a feature where:
  * the expiration date is extracted from the certificate itself
  * the UI shows a warning when we approach to the expiration date and eventually sends a notification mail
  * the UI shows a big fat warning when the certificate is expired and eventually sends a notification mail

However we don't have any of these requirements right now, let's keep the codebase as small as possible.